modules/lvms-restoring-volume-snapshots.adoc

```diff
@@ -0,0 +1,70 @@
+// Module included in the following assemblies:
+//
+// storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="lvms-restoring-volume-snapshots_{context}"]
+= Restoring volume snapshots
+
+To restore a volume snapshot, you must create a `PersistentVolumeClaim` custom resource (CR) with the `dataSource.name` field set to the name of the volume snapshot. 
```

Comment:
```suggestion
To restore a volume snapshot, you must create a `PersistentVolumeClaim` with the `dataSource.name` field set to the name of the volume snapshot. 
```
`PersistentVolume` and `PersistentVolumeClaim` are native objects so no custom resources :)

Reply:
Updated as per the suggestion.